### ISSUE TITLE
Nvidia CUDA "scale_npp" filter

### ISSUE THREAD

**Issue by egorapolonov** [bytedeco/javacpp-presets#857]
Hi @saudet ,

I'm using ffmpeg.class wrapper for reaching completely GPU-based transcoding.

Actually, I'm trying to repeat command which described here in tutorial https://devblogs.nvidia.com/nvidia-ffmpeg-transcoding-guide/
```bash
ffmpeg -vsync 0 –hwaccel cuvid -c:v h264_cuvid -i input.mp4 \ -c:a copy –vf scale_npp=1280:720 -c:v h264_nvenc -b:v 5M output_720.mp4
```
I see errors that there is no "scale_npp" filter. The ffmpeg should be configured with "--enable-libnpp".
Are there any maven-modules which contains libnpp. I'd like to include them into JavaCV

**Comment by saudet** [comment]
Yes, NPP is being bundled in the presets for CUDA, so it would be possible
to update the presets for FFmpeg with a "-gpu" extension and use it that
way.

**Comment by devjeonghwan** [comment]
Create and initialize `AVFilterContext` using `nppscale_init` and then define in, out frame informations(pixel format, size, etc..) to `NPPScaleStageContext`.

Now, you can call npp resize function like this.
```
static int nppscale_resize(AVFilterContext *ctx, NPPScaleStageContext *stage, AVFrame *out, AVFrame *in)
```

Let's check `vf_scale_npp.c` in [ffmpeg](https://www.ffmpeg.org/doxygen/3.2/vf__scale__npp_8c_source.html).`
